hasher-matcher-actioner/hmalib/metrics/query.py

```diff
@@ -0,0 +1,87 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+
+import functools
+from dataclasses import dataclass
+from enum import Enum
+import typing as t
+import boto3
+from datetime import datetime, timedelta
+
+from . import measure_performance, METRICS_NAMESPACE
```

Comment:
nit: should these be changed to cached methods not raw variables set in __init__.py?

Reply:
Do not see either changing during the lifetime of a lambda. So, could go either way. 

Leaving as is until ☝️ changes.